versions/spack.ver

```diff
@@ -1,4 +1,4 @@
-export spack_stack_ver=1.9.1
+export spack_stack_ver=1.9.2
 export spack_env=ue-oneapi-2024.2.1
 
 export cmake_ver=3.30.2
```

Comment:
Also, sorry I missed this before, but I think 3.30.2 is the system version of CMake on Ursa.  This probably hasn't caused a failure because the build.<host>.ver files all specify their own cmake versions.  The spack-stack version is 3.27.9.  I looked through the other version definitions and I think they are correct.